Fix powershell by adding verbose and removing brace

bagofmice opened the pull request:
Added -verbose so that you can grab the operations guid.
Removed dangling brace in powershell

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

MonicaRush commented:
Guillermo -- Thanks very much for your contribution. Please update the freshness date in the metadata section of the article next time you update an article. You can email me at monicar@microsoft.com if you have any questions. Thanks again.

gbellmann commented:
Thanks Monica. Will do next time!

MonicaRush commented:
You are welcome. Thanks again for your good work.

bagofmice committed:
Fix powershell by adding verbose and removing brace

Added -verbose so that you can grab the operations guid.
Removed dangling brace in powershell
